src/XMLimport.cpp

```diff
@@ -931,7 +931,7 @@ void XMLimport::readHostPackage(Host* pHost)
             break;
         } else if (isStartElement()) {
             if (name() == "name") {
-                pHost->mHostName = readElementText();
+                // name is intentionally ignored as it is already populated from the profile folder
```

Comment:
Some of the instability (*unstable* in the manner of *sweaty* TNT!) might be ameliorated by using `(void) Host::setName(const QString&)` here - I put it in precisely to try and propagate setting the `Host` member value to other classes that keep their own copy of it...

Reply:
Hmm, point taken, although the general idea stands - we should not have multiple source of truth for the same piece of information.

Reply:
How about reading the text and putting out a `qDebug()` warning if the returned value does not match the existing value - without trying to use it?

Reply:
We could... but then what is the warning gonna mean? It might or might not be intended and I don't see a situation where it'd be a problem.

Reply:
Well it will happen if the profile is renamed - or copied from somewhere *else* - and then, presumably it is intended, unless it isn't.